db/models.py

```diff
@@ -1 +1,33 @@
 from django.db import models
+
+
+class Race(models.Model):
+    STATUS_CHOICES = [
+        ("elf", "Elf"),
+        ("dwarf", "Dwarf"),
+        ("human", "Human"),
+        ("ork", "Ork"),
+    ]
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+    race = models.CharField(max_length=20, choices=STATUS_CHOICES)
```

Comment:
The `race` field with `STATUS_CHOICES` in the `Race` model is not required by the task and should be removed to align with the requirements.